WebService/v1/Controllers/DevicePropertiesController.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Filters;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Controllers
+{
+    [Route(Version.PATH + "/[controller]"), TypeFilter(typeof(ExceptionsFilterAttribute))]
+    public class DevicePropertiesController : Controller
+    {
+        private readonly IDeviceModels deviceModelsService;
+
+        public DevicePropertiesController(IDeviceModels deviceModelsService)
+        {
+            this.deviceModelsService = deviceModelsService;
+        }
+
+        [HttpGet]
+        public DevicePropertiesApiModel GetDevicePropertiesAsync()
```

Comment:
This API doesn't really return the current properties of a device, it returns the initial Device Model Properties. This could be confused with the Simulated Device properties, which can be constantly changing.

Will consumers call this thinking they will get the current state of a simulated device?

Consider renaming this to something like "InitialDeviceProperties"

Reply:
Since we made the API /v1/devicemodels/deviceproperties, this should be clear now.